src/formatter/helpers/issue_helpers.js

```diff
@@ -171,3 +173,15 @@ export function formatIssue({
   })
   return `${text}\n`
 }
+
+function getRetryWarningText(testCase, flakyColorFn) {
+  const result = testCase.result
+  if (!testCase.result) {
+    return ''
+  }
+  const attemptNumber = testCase.attemptNumber
+  if (attemptNumber > 1 || result.status === Status.RETRY) {
```

Comment:
Again don't think we need to test the attemptNumber.

Reply:
Same as previous reply: `attemptNumber` may no longer be needed.